api/config/cdr_config_stable.json

```diff
@@ -103,6 +103,24 @@
       "hasFitbitData": true,
       "hasCopeSurveyData": true,
       "accessTier": "registered"
+    },
+    {
+      "cdrVersionId": 7,
+      "isDefault": true,
+      "name": "Synthetic Dataset in the Controlled Tier",
+      "bigqueryProject": "fc-aou-cdr-synth-stable",
+      "bigqueryDataset": "test_R2019q4r3",
```

Comment:
I'm assuming the R here doesn't matter b/c it's synthetic but I just want to confirm

Same for `cdrDbName`

Reply:
Yeah, I was wondering the same. It's at least "correct" for now as it mirrors the CT config in lower environments. I think the reality is, we don't have a different CT-specific synthetic CDR. In an ideal world, we would, since there are differences in the data. Perhaps further down the line.